### DISCUSSION TITLE
Do not move 'worklet' from function to top of file

### DISCUSSION THREAD

**Discussion by Elmatoss** [trivago/prettier-plugin-sort-imports#223] [Q&A]
Hey guys, I'm working with react-native and reanimated. 

In order to declare a worklet function, I have to write the string 'worklet' at the start of the function

`const someFunction = () => {
    'worklet'
}
`

Problem: pretter-plugin-sort-imports always moves the 'worklet' string to the top of the file. And I can't disable prettier for that line or that entire file. Didn't find anything to disable the plugin for certain files neither. 

Any idea how to fix this?

Had to remove the plugin altogether for now :(

**Comment by zhuanghongji** [comment]
> Hey guys, I'm working with react-native and reanimated.
> 
> In order to declare a worklet function, I have to write the string 'worklet' at the start of the function
> 
> `const someFunction = () => { 'worklet' } `
> 
> Problem: pretter-plugin-sort-imports always moves the 'worklet' string to the top of the file. And I can't disable prettier for that line or that entire file. Didn't find anything to disable the plugin for certain files neither.
> 
> Any idea how to fix this?
> 
> Had to remove the plugin altogether for now :(

@Elmatoss Try downgrading to version 3.4.0, as it worked for me.
